contessa/db.py

```diff
@@ -53,6 +53,34 @@ def ensure_table(self, table: Table):
         table.create(bind=self.engine, checkfirst=True)
         logging.info(f"Created table {table.name}.")
 
+    def upsert(self, objs):
+        logging.info(f"Upserting {len(objs)} results.")
+
+        data = []
+        for o in objs:
+            a = o.__dict__.copy()
+            a.pop("_sa_instance_state", None)
```

Comment:
Can you please provide comment, why is this property filled and what is wrong with it?

Reply:
its one of the way on how to do model -> dict. the `_sa_instance_state` is just sqlalchemy internal thing, therefore not a column.

can add a short comment.